recipes/tvb-framework/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = 'tvb-framework' %}
+{% set version = '1.5.5' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  md5: 3f501c3f43705440f8953bf24aa6b0e5
+  sha256: 81c6b293bff080f4b52ec2c9ece5a3e109c951eb9dd42b1d90aed00589d4828b
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/tvb-framework/{{ name }}-{{ version }}.tar.gz
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    
+  run:
+    - python
+    - allensdk
+    - apscheduler
```

Comment:
apscheduler is no longer necessary.
Same for other dependencies. Please update this list according to the latest file here:
https://github.com/the-virtual-brain/tvb-framework/blob/master/setup.py
Attention: do not add dependencies already included as tvb-library dependencies

Reply:
Update it, and remove all that are already included in tvb-library